### PR TITLE
produce beta app in PRs

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-416-11404185aa74ac91388366542244066d7e9a9b61-armv7.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-416-11404185aa74ac91388366542244066d7e9a9b61-x86.apk)

File: src/app/main.cpp [hunk 1]
```diff
@@ -71,7 +71,8 @@ int main( int argc, char ** argv )
   // Set up the QSettings environment must be done after qapp is created
   QCoreApplication::setOrganizationName( "OPENGIS.ch" );
   QCoreApplication::setOrganizationDomain( "opengis.ch" );
-  QCoreApplication::setApplicationName( "QField" );
+  qDebug() << APP_NAME;
```
Comment (m-kuhn) [inline src/app/main.cpp]: Leftover?
If it's meaningful to leave, better with some context like `APP_NAME " is starting"`

**Comment by m-kuhn** [comment]
Nice one

**Comment by m-kuhn** [comment]
Superseded by #582

**Pull request closed without merging**